### PR TITLE
qmp_command: modify qmp_query-mice test to work on aarch64

### PR DESCRIPTION
**Pull request by rayx**

### CODE REVIEW

**Comment by rayx** [comment]
Note this depends on PR 1781 in avocado-vt:

https://github.com/avocado-framework/avocado-vt/pull/1781

File: qemu/tests/cfg/qmp_command.cfg [hunk 1]
```diff
@@ -25,6 +25,10 @@
                 cmd_return_value = "u'name': u'QEMU PS/2 Mouse';u'name': u'QEMU USB Tablet'"
             pseries:
                 cmd_return_value = "u'name': u'QEMU HID Tablet'"
```
Comment (vivianQizhu) [inline qemu/tests/cfg/qmp_command.cfg]: `pseries, arm64-pci, arm64-mmio:`?

Comment (rayx) [reply qemu/tests/cfg/qmp_command.cfg]: @vivianQizhu Changed.

**Comment by jingzhao84** [comment]
LGTM
Acked: jinzhao@redhat.com

**Comment by rayx** [comment]
@vivianQizhu FYI. Now that https://github.com/avocado-framework/avocado-vt/pull/1781 has been merged, this PR is ready for merge. I tested the code change on my aarch64 system a while back. Thanks.

**Comment by rayx** [comment]
@yanan-fu Is it about string format? I don't use Python3 much. I may take a look at this sometime next week.

**Comment by yanan-fu** [comment]
@rayx  Yes,  old design make 'u' as a string, which it not correct.
```
>>> a = u'anything'
>>> b = 'anything'
>>> bool(a == b)
True

>>> a = "u'anything'"
>>> b = "anything"
>>> bool(a == b)
False
```
I just send a patch https://github.com/autotest/tp-qemu/pull/1568 to make it work on python3. 

How do you think about add the change in this patch to mine ?

**Comment by vivianQizhu** [comment]
Thanks @yanan-fu I think this can be handled separately, since they are independent issues, this one only aim to arm support. I prefer merge this first, which might require you rebase your patch, is that okay for you?

**Comment by yanan-fu** [comment]
Sure, it is ok for me too.

**Comment by vivianQizhu** [comment]
Thanks all.

**Pull request merged**